client/script/client.js

```diff
@@ -67,14 +72,15 @@ function validate(tables, callback) {
 /**
  * All inputs are valid. Construct JSON objects and send them to the server.
  */
-function construct_and_send(tables, la, userkey) {
+function construct_and_send(tables, la) {
   // Start loading animation
   la.start();
 
   // Begin constructing the data
   var data_submission = { questions: {} };
 
   var session = $('#sess').val().trim();
```

Comment:
Actually I only missed this one, the previous #session is correct on the master branch